### PR TITLE
Strip quotes from translations via _()

### PR DESCRIPTION
**Pull request by tsouvarev**
If you have translation in template like _("Fox"), then django-babel will extract string with quotes: "Fox". This pull-request strips quotes from message, so extracted string will be Fox

### CODE REVIEW

**Comment by willkg** [comment]
What's this about? There's no explanation of the problem these commits are solving anywhere.

**Comment by tsouvarev** [comment]
If you have translation in template like `_("Fox")`, then django-babel will extract string with quotes: `"Fox"`. This pull-request strips quotes from message, so extracted string will be `Fox`

**Comment by tsouvarev** [comment]
Any feedback?

**Comment by graingert** [comment]
@tsouvarev can you move the link fixes to another PR, please?

File: django_babel/extract.py [hunk 1]
```diff
@@ -10,6 +10,14 @@
 from django.utils.encoding import smart_text
 
 
+def strip_quotes(s):
```
Comment (graingert) [inline django_babel/extract.py]: From stack overflow:
http://stackoverflow.com/a/20577580/833093
```
def dequote(s):
    """
    If a string has single or double quotes around it, remove them.
    Make sure the pair of quotes match.
    If a matching pair of quotes is not found, return the string unchanged.
    """
    if (s[0] == s[-1]) and s.startswith(("'", '"')):
        return s[1:-1]
    return s
```

**Comment by tsouvarev** [comment]
https://github.com/python-babel/django-babel/pull/31
PR with fixes to docs

**Comment by tsouvarev** [comment]
Do you need anything else?

**Comment by graingert** [comment]
@tsouvarev ideally 00e92e7 would be a separate PR

**Pull request merged**